sentry_sdk/integrations/logging.py

```diff
@@ -110,17 +110,25 @@ def _breadcrumb_from_record(record):
     # type: (LogRecord) -> Dict[str, Any]
     return {
         "type": "log",
-        "level": _logging_to_event_level(record.levelname),
+        "level": _logging_to_event_level(record),
         "category": record.name,
         "message": record.message,
         "timestamp": datetime.datetime.utcfromtimestamp(record.created),
         "data": _extra_from_record(record),
     }
 
 
-def _logging_to_event_level(levelname):
-    # type: (str) -> str
-    return {"critical": "fatal"}.get(levelname.lower(), levelname.lower())
+def _logging_to_event_level(record):
+    # type: (LogRecord) -> str
+    default = 'error'
+    return {
+        logging.DEBUG: 'debug',
+        logging.INFO: 'info',
+        logging.WARNING: 'warning',
```

Comment:
that's just an alias:

>>> import logging
>>> logging.WARN
30
>>> logging.WARNING
30
>>>

Reply:
Oh, but so is critical and fatal.

It will try to make a duplicate in the dict, but I guess we can just let python ignore that, like we do for fatal/critical.